doc: CSS tweaks

XhmikosR opened the pull request:
* split selectors to one per line
* remove units from 0
* use double colon pseudo selectors consistently
* use `background-color` instead of the `background` shorthand since only the color is supposed to change
* remove leading zeros from numbers
* remove redundant values from shorthands
* use lowercase hex for colors
* use numeric values for `font-weight`

<!--
Thank you for your pull request. Please provide a description above and review
the requirements below.

Bug fixes and new features should include tests and possibly benchmarks.

Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
-->

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

<!--
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
-->

XhmikosR commented on doc/api_assets/style.css:
```diff
@@ -151,39 +164,40 @@ ol.version-picker li:last-child a {
 .api_stability {
-  color: white !important;
-  margin: 0 0 1rem 0;
-  font-family: "Lato", "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", Verdana, Tahoma, sans-serif;
```
this isn't needed because it's inherited by the `body` font-family.

Trott commented:
@nodejs/collaborators  This could use some reviews from some web/CSS-savvy folks.

Trott commented:
Lite CI: https://ci.nodejs.org/job/node-test-pull-request-lite-pipeline/3952/

ronkorving commented:
As a not-so-web-developer lately, how much of this is taste, and how much is objectively better? I'm just asking because I don't think we should accept PRs that flip code style based on personal taste alone, unless it's been discussed.

XhmikosR commented:
Just to clarify, the only style change is splitting the selectors to one per line. Which is similar to using one var in JS.

I personally find this much easier to read.

Note that these changes aim to make the CSS consistent first, and following the best practices nowadays.

antsmartian commented:
This seems to be just applying consistent patterns across our css files . It would be great, what patterns do people from https://github.com/nodejs/nodejs.dev follow. Since we use handful of css over there in nodejs.dev, making the patterns / css practice consistent across repo will be really helpful.

XhmikosR commented:
Note that I have already switched the current website to [Sass](https://github.com/nodejs/nodejs.org/pull/2530) and also added Stylelint, which means we can probably just create a stylelint config later for use in most nodejs projects.

Trott commented:
Landed in cdf8f70
